sentences/de/_common.yaml

```
@@ -9,12 +9,144 @@ responses:
     # Errors for when user is not logged in
     no_area: "Kein Bereich mit dem Namen {{ area }} vorhanden"
     no_floor: "Keine Etage mit dem Namen {{ floor }} vorhanden"
-    no_domain: "Kein {{ domain }} vorhanden"
-    no_domain_in_area: "Im Bereich {{ area }} ist kein {{ domain }} vorhanden"
-    no_domain_in_floor: "In der Etage {{ floor }} ist kein {{ domain }} vorhanden"
-    no_device_class: "Kein {{ device_class }} vorhanden"
-    no_device_class_in_area: "Im Bereich {{ area }} ist kein {{ device_class }} vorhanden"
-    no_device_class_in_floor: "In der Etage {{ floor }} ist kein {{ device_class }} vorhanden"
+    no_domain: |
+      {% set translations = {
+        "button": "keine Taste",
+        "camera": "keine Kamera",
+        "input_button": "keine Tasteneingabe",
+        "alarm_control_panel": "keine Alarmzentrale",
+        "automation": "keine Automatisierung",
+        "fan": "kein Ventilator",
+        "climate": "kein Thermostat",
+        "humidifier": "kein Luftbefeuchter",
+        "input_boolean": "keine Boolesche Eingabe",
```

```suggestion
        "input_boolean": "kein Schalter",
```

Boolesche Eingabe is not common

---

yeah, I was struggeling with it, too - but i've just taken the translation from lokalize, since this is already present in the UI and those should be common to the user

---

maybe we can use "Helfer Schalter", since only "Schalter" is already the `switch` domain?

---

Yeah far better then Boolesche :+1: